### DISCUSSION TITLE
Option for setting asset field title setting to textarea

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#8812] [Ideas]
I would like to use built-in `title` field of an asset, but its standard input which has only one line. This is not practical for longer text. Why not switch that to textarea, or add config setting which would set this field to textarea?

**Comment by brandonkelly** [comment]
I’ll leave this open, but it seems a little crazy to me :) A title is a title, and titles should generally be as short as they can be, to make the asset easy to identify throughout the system.

You can always add a Plain Text field to your volume’s field layout, if you want to store a longer description that way.

**Comment by piotrpog** [comment]
@brandonkelly Title field has this advantage over custom field, that its contents are shown on files list. If each file needs some description and this description can only be viewed and reviewed when opening the asset details window, it quickly gets cumbersome with large number of files in asset field.

**Comment by brandonkelly** [comment]
You can show Plain Text fields’ values on element index pages. Click the “Customize” link below the source list, and then tick the checkmark next to your description field.
